index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="./css/style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Lato:wght@900&family=Poppins:wght@100;300&display=swap"
+    rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference</title>
+</head>
+
+<body>
+  <section class="navbar">
+    <div class="menu-icon">
+      <hr class="m-menu">
+      <hr class="m-menu">
+      <hr class="m-menu">
+    </div>
+    <ul class="menu-ul">
+      <div class="head-line">
+        <img alt="facebook" class="head-img" src="./Images/facebook.png">
+        <img alt="facebook" class="head-img" src="./Images/tweeter.png">
+        <li class="head-link"> English </li>
+        <li class="head-link"> Mypage</li>
+        <li class="head-link">Logout</li>
+      </div>
```

Comment:
- [OPTIONAL] Super cool The top short bar section with social media links, logout, and language links at the top. Well done :+1: However, I think you can improve the style on this section  :grinning: 

Let's improve it :point_down: 

 ![](https://camo.githubusercontent.com/96bd23868b8867313e32b0dd0bd61c02273f1ccfff54263caef020c9b2a779c2/68747470733a2f2f692e696d6775722e636f6d2f6e733846646b6b2e706e67)